### ISSUE TITLE
Kubectl outputs unecessary empty line

### ISSUE THREAD

**Issue by prary** [kubernetes/kubectl#766]
kubectl version 1.15.x

```
[root@iaasn00126855 out]# kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3+vmware.1", GitCommit:"23e2c41321122d2e0c20eb0111a8abedfbf0fd41", GitTreeState:"clean", BuildDate:"2019-08-19T18:44:31Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.3+vmware.1", GitCommit:"23e2c41321122d2e0c20eb0111a8abedfbf0fd41", GitTreeState:"clean", BuildDate:"2019-08-19T18:42:30Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
```
Kubectl throws extra line only when ```get all``` is performed.

```
[root@test out]# kubectl get all -n 103000-pra-dev
NAME               READY   STATUS    RESTARTS   AGE
pod/kaniko-mn785   1/1     Running   0          79s








[root@test out]# 
````

**Comment by eddiezane** [comment]
I'm able to reproduce this on 1.15.x (up to the recently released 1.15.6) but it appears fixed on 1.16.x.

**Comment by seans3** [comment]
Since this is fixed in 1.16, closing this. (It will not be backported, since its low priority).

/close
